cli/src/commands/apply/resources.ts

```diff
@@ -90,7 +91,11 @@ Uploaded successfully!
         // Client-side validation can be added later on to not require a server hop
         switch (kind) {
           case 'Schema':
-            rg.schemas!.push(resource);
+            if (resource.schema) {
```

Comment:
Throw error if it has neither?

Reply:
We don't throw error on other resources.. 
I think the error will be thrown either by `graphql-request` or by Stitch Registry.